osm2gtfs.py

```diff
@@ -298,5 +293,34 @@ def load_config(file):
 
     return config;
 
+def load_agency(config):
+    '''
```

Comment:
Great you are adding documentation already! Docstrings should be made with double quotes `"` btw.